examples/Scaffolding/scaffolding.md

```diff
@@ -1,5 +1,8 @@
 # Automating Team Project set up
 
+In any team, project, or an enterprise there is a need to templatize and automate processes associated with a project. E.g.
```

Comment:
`enterprise` ==> `organization`..
not sure if this is targeted to our org concept or customers in general

Reply:
Resolved